assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -143,11 +149,11 @@ cmdline_isolation=+isolcpus=managed_irq,${isolated_cores}
 {{end}}
 
 {{if .RealTimeHint}}
-cmdline_realtime=+nohz_full=${isolated_cores} tsc=reliable nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11
+cmdline_realtime=+nohz_full=${isolated_cores} nosoftlockup skew_tick=1 rcutree.kthread_prio=11
 {{end}}
 
 {{if .HighPowerConsumption}}
-cmdline_power_performance=+processor.max_cstate=1 intel_idle.max_cstate=0
+cmdline_power_performance=${high_power_consumption_cstate}
 {{end}}
 
 {{if and .HighPowerConsumption .RealTimeHint}}
```

Comment:
I don't think ARM supports the `idle=poll` argument (see https://github.com/torvalds/linux/blob/42f7652d3eb527d03665b09edac47f85fb600924/Documentation/admin-guide/kernel-parameters.txt#L2001). Maybe the best approach would be just to reject the .HighPowerConsumption hint for ARM servers for now since we have no way support it - is there a way to do that?

Reply:
Good catch, does not look supported on arm so we should only set it for x86.

Reply:
@MarSik I've used composition for this like on the other changes